components/bandwidth/bandwidth.app.js

```diff
@@ -1,4 +1,59 @@
+const bandwidthMessaging = require("@bandwidth/messaging");
+
 module.exports = {
   type: "app",
   app: "bandwidth",
+  propDefinitions: {
+    messageTo: {
+      type: "string",
+      label: "To",
+      description: "The number the message will be sent to, in E164 format ex `+19195551234`.",
+    },
+    from: {
+      type: "string",
+      label: "From",
+      description: "The number the call or message event will come from, in E164 format ex `+19195551234`.",
+    },
+    message: {
+      type: "string",
+      label: "Message",
+      description: "The text message content.",
+    },
+    messagingApplicationId: {
+      type: "string",
+      label: "Messaging Application ID",
+      description: "The ID from the messaging application created in the [Bandwidth Dashboard](https://dashboard.bandwidth.com).\n\nThe application must be associated with the location that the `from` number lives on.",
```

Comment:
Is there a Bandwidth API for retrieving this value from the user's account?

Reply:
> would you rather us build that logic in as opposed to asking the user for the application ID?

If possible, yes. See my comment above re: the "From" prop. Generally, this dramatically improves the usability of components.

If you can pull this behind the scenes, it probably doesn't even need to be a prop, which would further simplify.